TheJumpCloud/support
# Sa 2642 get jc command search params

## Issues
* [SA-2642](https://jumpcloud.atlassian.net/browse/sa-2642) - SA-2642-Get-JCCommand-Search-Params

## What does this solve?
Add search functionality to Get-JCCommand
## Is there anything particularly tricky?
N/A
## How should this be tested?
1. Import this branch's module
2. Check if all these search parameters work:  name, command, commandType, launchType, listensTo, schedule, trigger, scheduleRepeatType, organization, _id, byId
